src/main/java/org/jabref/gui/ClipBoardManager.java

```diff
@@ -165,4 +166,19 @@ public void setContent(List<BibEntry> entries, BibEntryTypesManager entryTypesMa
         clipboard.setContent(content);
         setPrimaryClipboardContent(content);
     }
+
+    public void setContent(List<BibEntry> entries, BibEntryTypesManager entryTypesManager, List<BibtexString> stringConstants) throws IOException {
+        final ClipboardContent content = new ClipboardContent();
+        BibEntryWriter writer = new BibEntryWriter(new FieldWriter(preferencesService.getFieldPreferences()), entryTypesManager);
```

Comment:
I see a lot of duplicated code with the other method. Please extract the common parts to a common method

Reply:
I have extracted the methods as suggested, please check them when convenient, thanks!